### PR TITLE
Add minimal support fot abstract classes

### PR DESCRIPTION
**Pull request by jonykalavera**
Made these changes to mark abstract classes as such in the diagram

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ python = "^3.8"
 
 [tool.poetry.dev-dependencies]
 pylint = "2.12.2"
-pytest = "6.2.2"
+pytest = "6.2.5"
```
Comment (jonykalavera) [inline pyproject.toml:23]: Had to update due to https://github.com/pytest-dev/pytest/discussions/9195

**Comment by lucsorel** [comment]
hi @jonykalavera, thank you for this pull-request :smiley: 

I made a few comments in the review.

Besides, there are a couple of other things to do, which I forgot to mention in the contributing guidelines:
- add yourself in the [contributions section of the README.md](https://github.com/jonykalavera/py2puml/blob/add-abstract/README.md#contributions) file. Suggestion:

```text
# Contributions

* [Luc Sorel-Giffo](https://github.com/lucsorel)
* [Doyou Jung](https://github.com/doyou89)
* [Julien Jerphanion](https://github.com/jjerphan)
* [Luis Fernando Villanueva Pérez](https://github.com/jonykalavera)
```

- add a changelog line in the [Changelog section of the README.md](https://github.com/jonykalavera/py2puml/blob/add-abstract/README.md#changelog) file.

```text
# Changelog

* `0.6.0`: handle abstract classes
* `0.5.4`: fixed the packaging so that the contribution guide is included in the published package
```

- update the version of the library to `0.6.0`. That needs to be done:
  - in the [version attribute of the pyproject.toml](https://github.com/jonykalavera/py2puml/blob/add-abstract/pyproject.toml#L3) file
  - in the [cli.py module](https://github.com/jonykalavera/py2puml/blob/add-abstract/py2puml/cli.py#L12) (the string value associated to the `version` parameter)
  - in the [test__init__.py](https://github.com/jonykalavera/py2puml/blob/add-abstract/tests/py2puml/test__init__.py#L5) file

That should not take long. Let me know if this is ok for you.

File: py2puml/py2puml.domain.puml [hunk 2]
```diff
@@ -6,6 +6,7 @@ class py2puml.domain.umlclass.UmlAttribute {
 }
 class py2puml.domain.umlclass.UmlClass {
   attributes: List[UmlAttribute]
+  abstract: bool
```
Comment (jonykalavera) [inline py2puml/py2puml.domain.puml:9]: I guess we should update this one too. right?

**Comment by jonykalavera** [comment]
> I made a few comments in the review.

I saw none other than the requested in https://github.com/lucsorel/py2puml/pull/27#issuecomment-1089306137
just checking.

**Comment by lucsorel** [comment]
> I saw none other than the requested in https://github.com/lucsorel/py2puml/pull/27#issuecomment-1089306137
> just checking.

if you scroll up the "Conversation" page, you should see the other review comments I made. Or you can browse the "Files changed" tab (https://github.com/lucsorel/py2puml/pull/27/files) to see them. My comments are related to the separation of concerns between the modeling classes (representing the UML model derived from the code inspection) and the generation of the PlantUML code: I am not fond of `UmlClass.item_type` returning `'abstract class'` or `'class'`; I suggested small changes to fix that.

There is also some assertions to add to the unit test.

**Comment by jonykalavera** [comment]
> if you scroll up the "Conversation" page, you should see the other review comments I made. Or you can browse the "Files changed" tab (https://github.com/lucsorel/py2puml/pull/27/files) to see them.
I honestly do not see any. by any chance did you miss submitting the review comments?
> My comments are related to the separation of concerns between the modeling classes (representing the UML model derived from the code inspection) and the generation of the PlantUML code: I am not fond of UmlClass.item_type returning 'abstract class' or 'class';  I suggested small changes to fix that.
I get it. let's fix it. :+1:

**Comment by lucsorel** [comment]
hi @jonykalavera 

> I honestly do not see any. by any chance did you miss submitting the review comments? at the moment the comment count (including this one) is at: 6

You were totally right, forgive me: I did not submit them. I am not familiar with the review process of github and did not realize that my comments were not yet visible to you. It should now be the case. Thank you for helping out :smiley:

**Comment by jonykalavera** [comment]
I think it's ready now. let me know if I missed anything. ✌️

**Review by lucsorel — APPROVED** [review]

**Comment by lucsorel** [comment]
the unit test `test_py2puml_model_on_py2uml_domain` does not pass anymore (`poetry run python -m pytest -v`). I am having a look at it...

**Review by lucsorel — APPROVED** [review]

**Comment by lucsorel** [comment]
your contribution is available https://pypi.org/project/py2puml/ :smiley: 

Thank you very much and sorry for the review process glitch!

**Pull request merged**